web-services/deploy/configuration/src/main/resources/datawave/query/QueryLogicFactory.xml

```diff
@@ -235,7 +235,7 @@
         </property>
         <property name="queryPlanner" ref="DefaultQueryPlanner" />
         <property name="sendTimingToStatsd" value="false" />
-	<property name="collectQueryMetrics" value="false" />
```

Comment:
@keith-ratcliffe @ivakegg Any feelings against having this set to true by default?